include/swift/SILOptimizer/Analysis/ARCAnalysis.h

```diff
@@ -117,7 +117,7 @@ valueHasARCDecrementOrCheckInInstructionRange(SILValue Op,
 /// in the predecessors. 
 ///
 /// The search stop when we encounter an instruction that may decrement
-/// the return'ed value, as we do not want to create a lifetime gap once the
+/// the returned value, as we do not want to create a lifetime gap once the
```

Comment:
I understand that `return` is a technical term, but I don't think this spelling added value here.